service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceController.kt

```diff
@@ -193,6 +194,9 @@ class ChildAttendanceController(
         @PathVariable childId: ChildId,
         @RequestBody body: ArrivalRequest
     ) {
+        if (body.arrived.second != 0 || body.arrived.nano != 0) {
+            throw BadRequest("Arrival time must be on the minute")
+        }
```

Comment:
having to check this everywhere feels suboptimal. 🤔 maybe there could at least be a shared extension function `LocalTime.assertMinutePrecision()`?

Reply:
The ultimate fix would be to have `LocalHm` in request and response types which would follow the _parse don't validate_ principle, but I didn't want to do that in this PR.

Reply:
... and the `minute == nano == 0` check is already there as a DB constraint, so these checks could actually just be removed because the frontend doesn't send seconds.

Reply:
Checks removed